### ISSUE TITLE
No version information on kubectl kcp plugim

### ISSUE THREAD

**Issue by matzew** [kcp-dev/kcp#1380]
**Describe the bug**

I am missing a `version` information on the `kubectl-kcp` plugin

**To Reproduce**

`kubectl-kcp --help` does not list a version

**Expected behavior**

**Additional context**

**Comment by apoorvajagtap** [comment]
Hello @stevekuznetsov , @varshaprasad96 , I think `--version` for `kubectl-kcp` is still missing after #1409, so I just raised another PR #1432 with the similar changes for kubectl-kcp. 

I have been just exploring a little, so it'd be great if someone can take a look. Let me know in case I am missing something... Thanks!
